docs/installation.md

```diff
@@ -12,20 +12,26 @@ next:
 
 overlay-kit is available on [npm](https://npmjs.com/package/overlay-kit) for Node.js and Bun.
 
-## Node.js
+## Prerequisites
+
+- [React](https://react.dev/) version 18 or higher.
```

Comment:
It's not higher. ^18 only cover major version 18. not 19 so In my opinion, this should be updated

Reply:
https://github.com/toss/overlay-kit/blob/main/packages/package.json#L60

Reply:
I agree about this, I will make update PR

Reply:
Thanks! ❤️

Reply:
Before making a fix, as @manudeli  mentioned, currently overlay-kit only covers react 18 version. 
I wonder if this will be a problem for specifying dependencies higher react 18 version.

It would be good to talk about whether **I should modify the docs** or modify of a package.json that covers dependencies `>= 18`

Reply:
In my opinion, `>= 18` meaning contain not only `19`, `20`, `30` so It would be better that `^18 || ^19` is enough or don't add it yet before releasing 19

Reply:
@manudeli plz check #23 , thanks!